example/lib/pages/background_playback.dart

```diff
@@ -0,0 +1,90 @@
+import 'dart:io';
+
+import 'package:audio_session/audio_session.dart';
+import 'package:bitmovin_player/bitmovin_player.dart';
+import 'package:bitmovin_player_example/controls.dart';
+import 'package:bitmovin_player_example/env/env.dart';
+import 'package:flutter/material.dart';
+
+/// This example showcases how to achieve background playback with the
+/// Bitmovin Player using the `audio_session` package.
+/// https://pub.dev/packages/audio_session
```

Comment:
We don't have comments on other samples but I thought it would make sense to add it.